test/testcafe/spec/ChallengeAuthenticatorOktaVerify_spec.js

```diff
@@ -15,116 +22,120 @@ const requestLogger = RequestLogger(/challenge/,
   const selectAuthenticatorPageObject = new SelectAuthenticatorPageObject(t);
   await selectAuthenticatorPageObject.navigateToPage();
+
+  await t.expect(selectAuthenticatorPageObject.getFormTitle()).eql(FORM_TITLE);
+  await t.expect(selectAuthenticatorPageObject.getFormSubtitle()).eql(FORM_SUBTITLE);
+  await t.expect(selectAuthenticatorPageObject.getFactorsCount()).eql(3);
+
   return selectAuthenticatorPageObject;
 }
 
-test.requestHooks(mockChallengeOVSelectMethod)('should load select method list with okta verify options', async t => {
+test.requestHooks(mockChallengeOVSelectMethod)('preserve the order of authenticator list ' +
+  'when the `deviceKnown` attribute is either null or false', async t => {
   const selectAuthenticatorPage = await setup(t);
-  await t.expect(selectAuthenticatorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
-  await t.expect(selectAuthenticatorPage.getFormSubtitle()).eql('Select from the following options');
-  //await t.expect(selectAuthenticatorPage.getFactorsCount()).eql(3);
-  await t.expect(selectAuthenticatorPage.getFactorLabelByIndex(0)).eql('Use Okta Verify on this device');
-  await t.expect(await selectAuthenticatorPage.factorDescriptionExistsByIndex(0)).eql(false);
-  await t.expect(selectAuthenticatorPage.getFactorIconClassByIndex(0)).contains('mfa-okta-verify');
-  await t.expect(selectAuthenticatorPage.getFactorSelectButtonByIndex(0)).eql('Select');
-
-  await t.expect(selectAuthenticatorPage.getFactorLabelByIndex(1)).eql('Get a push notification');
-  await t.expect(await selectAuthenticatorPage.factorDescriptionExistsByIndex(1)).eql(false);
-  await t.expect(selectAuthenticatorPage.getFactorIconClassByIndex(1)).contains('mfa-okta-verify');
-  await t.expect(selectAuthenticatorPage.getFactorSelectButtonByIndex(1)).eql('Select');
-
-  await t.expect(selectAuthenticatorPage.getFactorLabelByIndex(2)).eql('Enter a code');
-  await t.expect(await selectAuthenticatorPage.factorDescriptionExistsByIndex(2)).eql(false);
-  await t.expect(selectAuthenticatorPage.getFactorIconClassByIndex(2)).contains('mfa-okta-verify');
-  await t.expect(selectAuthenticatorPage.getFactorSelectButtonByIndex(2)).eql('Select');
+
+  await verifyFactorByIndex(t, selectAuthenticatorPage, 0, PUSH_NOTIFICATION_TEXT);
+  await verifyFactorByIndex(t, selectAuthenticatorPage, 1, ENTER_CODE_TEXT);
+  await verifyFactorByIndex(t, selectAuthenticatorPage, 2, FAST_PASS_TEXT);
 
   // signout link at enroll page
   await t.expect(await selectAuthenticatorPage.signoutLinkExists()).ok();
-  await t.expect(selectAuthenticatorPage.getSignoutLinkText()).eql('Back to sign in');
+  await t.expect(selectAuthenticatorPage.getSignoutLinkText()).eql(SIGN_OUT_TEXT);
 });
 
-test.requestHooks(mockChallengeOVSelectMethod)('should load select method list with okta verify and no sign-out link', async t => {
+test.requestHooks(mockChallengeOVSelectMethod)('should load select method list with okta verify and no sign link', async t => {
```

Comment:
let's keep `sign-out` link. I know the text does not say "sign out" anymore, but it is still that idea... you could also say "no cancel link" if you prefer

Reply:
I reverted back to `sign-out`